playground/backend/logging.properties

```diff
@@ -0,0 +1,5 @@
+handlers=java.util.logging.FileHandler
+
+java.util.logging.FileHandler.pattern={logFilePath}
+java.util.logging.FileHandler.count=1
+java.util.logging.FileHandler.formatter=java.util.logging.SimpleFormatter
```

Comment:
Could you please add an empty line at the end of this file?

Reply:
Done.